contributors/design-proposals/auth/pod-security-policy.md

```diff
@@ -156,10 +156,10 @@ type VolumeSecurityPolicy struct {
 	// More info: http://kubernetes.io/docs/user-guide/persistent-volumes/#persistentvolumeclaims
 	PersistentVolumeClaim bool `json:"persistentVolumeClaim,omitempty"`
 	// RBD allows or disallows the use of the RBD volume plugin.
-	// More info: http://releases.k8s.io/HEAD/examples/volumes/rbd/README.md
+	// More info: https://github.com/kubernetes/examples/blob/master/volumes/rbd/README.md
 	RBD bool `json:"rbd,omitempty"`
 	// Cinder allows or disallows the use of the Cinder volume plugin.
-	// More info: http://releases.k8s.io/HEAD/examples/mysql-cinder-pd/README.md
+	// More info: https://github.com/kubernetes/examples/blob/master/mysql-cinder-pd/README.md
```

Comment:
You have a couple other PRs that are also touching these links (https://github.com/kubernetes/community/pull/4268 and https://github.com/kubernetes/community/pull/4269) and also do some changes like https -> http -> https again. If you could, please consolidate these down to a single PR and squash the commits down 👍